### PR TITLE
PMM-4019 custom labels presentation

### PR DESCRIPTION
**Pull request by darly-s**
https://jira.percona.com/browse/PMM-4019

### CODE REVIEW

File: src/app/inventory/nodes-table/models/generic.model.ts [hunk 1]
```diff
@@ -10,8 +13,14 @@ export class GenericModel {
   isDeleted: boolean;
 
   constructor(params, type) {
+    params.custom_labels = {
+      mongokey: 'value1',
+      mongokey2: 'value2',
```
Comment (askomorokhov) [inline src/app/inventory/nodes-table/models/generic.model.ts]: what do these values mean?

**Comment by askomorokhov** [comment]
@darly-s, please resolve conflict.

File: src/app/inventory/nodes-table/models/generic.model.ts (hunk 1 above)
Comment (rnovikovP) [reply src/app/inventory/nodes-table/models/generic.model.ts]: Oh  .. @darly-s  this was just an example from my  test case

Comment (darly-s) [reply src/app/inventory/nodes-table/models/generic.model.ts]: Yes, sorry, it need to be removed) It was code to test output

**Pull request merged**